### ISSUE TITLE
How to use the converted model for prediction?

### ISSUE THREAD

**Issue by bshao001** [amir-abdi/keras_to_tensorflow#13]
@amir-abdi 
Firstly, thank you for sharing this great work. However, I don't see any good example to demonstrate the usage for prediction using the converted TensorFlow model (.pb file). With Keras, you have predict function from the model,  while with a normally trained TF model, you specified the logits and placeholder for the input. With this converted model, I have no idea of how to perform the prediction. Could you please give me some guidance or a concrete example? Thank you very much!

**Comment by bshao001** [comment]
Never mind. I have figured out everything.

**Comment by amir-abdi** [comment]
I liked your suggestion and started to put together a sample deployment code for the readme. If you have figured it out and wanted to contribute to readme, feel free to initiate a pull request.

**Comment by bshao001** [comment]
@amir-abdi Thanks for your reply.

I have just uploaded a cleaner version on my git account. It is a working example. I added some extra comments to explain all the details. You can refer that to create samples if you want.

https://github.com/bshao001/DmsMsgRcg/blob/master/textdect/convertmodel.py
